libr/bin/p/bin_coff.c

```diff
@@ -113,19 +113,24 @@ static bool _fill_bin_symbol(RBin *rbin, struct r_bin_coff_obj *bin, int idx, RB
 		ptr->type = r_str_constpool_get (&rbin->constpool, sdb_fmt ("%i", s->n_sclass));
 		break;
 	}
+	ptr->vaddr = ptr->paddr;
 	ptr->size = 4;
 	ptr->ordinal = 0;
 	return true;
 }
 
+static bool is_imported_symbol(struct coff_symbol *s) {
```

Comment:
inline

Reply:
not sure it's really necessary... the compiler will inline this anyway if possible, when you compile in release mode.